ERCS/erc-7699.md

```diff
@@ -0,0 +1,90 @@
+
+## Characteristics
+
+The tokens are confined solely to minting and burning functionalities, precluding any transferability.
+During the execution of minting and burning transactions, the system logs the cumulative reputation associated with the respective account.
+On-chain events are deployed to meticulously capture and document the progression of reputation accumulation.
+
+## Specification
+
+Notes:
+The contract is designed to be compiled with a Solidity compiler version greater than or equal to 0.8.20 but less than 0.9.0.
+
+The smart contract realising this EIP mandate must fully incorporate all methods prescribed by [ERC-20](./erc-20.md), with the exception of transfer operations. Furthermore, exclusivity to implementing solely this EIP is required.
+
+## Token Methods
+
+### Internal _update Function Override
+
+```solidity
+function  _update(address from, address to, uint256 value) internal virtual override { 
+    if (from != address(0) && to != address(0)) {
+        revert TransferNotAllowed();
+    }
+    super._update(from, to, values);
+}
+```
+
+
+Description: This function overrides the _update function inherited from the [ERC-20](./erc-20.md) contract. It is an internal function that is used internally by the [ERC-20](./erc-20.md) contract to update the balances of token holders during transfer operations. In this contract, it restricts transfers between non-zero addresses, reverting the transaction with the "TransferNotAllowed" error if both the sender and receiver addresses are non-zero.
+
+### Mint
+
+```solidity
+function mint(address to, uint256 amount) public {
+    _mint(to, amount);
+}
+```
+
+Description: This function allows the contract owner to mint new tokens and allocate them to a specified address. It increases the balance of the specified address by the given amount of tokens. This function can be used to create new tokens and distribute them as needed.
+
+### Burn
+
+```solidity
+ function burn(address from, uint256 amount) public {
+    _burn(from, amount);
+}
+```
+
+Description: This function allows token holders to burn a specific amount of their tokens. It decreases the balance of the specified address by the given amount of tokens. Burning tokens means removing them from circulation permanently. This function provides token holders with the ability to reduce their token balances voluntarily.
+
+## Backwards Compatibility
+
+This proposal is only partially compatible with [ERC-20](./erc-20.md) because it makes tokens non-transferable after the first transfer.
+
+## Citation
+
+Please cite this document as:
+Burve Labs. "Bridging the Gap: Soul Resonance Token (SRT)  Non-transferable yet Tradable Asset Type based on Bonding Curve"Burve Labs. 20 Apr. 2024.
+
+## References
+
+**Implementations**
+
+- [ERC-7699 Reference Implementation](https://github.com/mike0x1024/srt-template)
+
```

Comment:
This is a non-standard section of EIPs. If you'd like to include a reference implementation, you may do so in the `assets` directory and link to it there.